Description: Upstream changes introduced in version 3.03-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 xpdf (3.03-1) UNRELEASED; urgency=low
 .
   * New upstream release:
     - Refresh patches.
     - Drop 12-fix-typo-in-help.patch, fix-437725.patch, fix-479467.patch, and
       fix-577031.patch (now applied upstream).
   * Use -fpermissive build flag.
   * Really pass all arguments when using -m.
   * Fix broken -z argument (closes: #636390).
 .
 The person named in the Author field signed this changelog entry.
Author: Michael Gilbert <michael.s.gilbert@gmail.com>
Bug-Debian: http://bugs.debian.org/636390

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- xpdf-3.03.orig/xpdf/GlobalParams.cc
+++ xpdf-3.03/xpdf/GlobalParams.cc
@@ -73,6 +73,11 @@ extern XpdfPluginVecTable xpdfPluginVecT
 #  endif
 #endif
 
+// implement xpdf 3.03 fopen wrapper
+extern FILE *openFile(const char *path, const char *mode) {
+    return fopen(path, mode);
+}
+
 //------------------------------------------------------------------------
 
 #define cidToUnicodeCacheSize     4
--- xpdf-3.03.orig/xpdf/GlobalParams.h
+++ xpdf-3.03/xpdf/GlobalParams.h
@@ -39,6 +39,8 @@ class GlobalParams;
 class SysFontList;
 class Stream;
 
+extern FILE *openFile(const char *path, const char *mode);
+
 //------------------------------------------------------------------------
 
 // The global parameters object.
